main/templates/main/base.html

```diff
@@ -1,5 +1,8 @@
 {% load static %}
 {% load django_bootstrap5 %}
+<link rel="shortcut icon"
+      type="image/png"
+      href="{% static 'main/images/Favicon.png' %}">
```

Comment:
Looks like this typically goes at the bottom of `<head>`, just before `</head>`. See e.g. [here](https://stackoverflow.com/questions/48354130/what-is-the-proper-place-of-favicon-references-in-head).